public/kibi_timeline.js

```diff
@@ -394,3 +401,4 @@ define(function (require) {
     } // end of link function
   });
 });
+
```

Comment:
The nesting in the loop requires the 2 spaces additional indentation for readability.